src/service/grpcweb.ts

```diff
@@ -186,12 +190,23 @@ function generateTypescriptDefinition(fileDescriptor: FileDescriptorProto, expor
       printer.printLn(`export class ${service.name} {`);
       printer.printIndentedLn(`static readonly serviceName: string;`);
       service.methods.forEach(method => {
-        printer.printIndentedLn(`static readonly ${method.name}: ${method.serviceName}${method.name};`);
+        printer.printIndentedLn(`static readonly ${method.nameAsPascalCase}: ${method.serviceName}${method.nameAsPascalCase};`);
       });
       printer.printLn(`}`);
       printer.printEmptyLn();
     });
 
+  printer.printLn(`export type ServerStreamEventType = 'data'|'end';`);
+  printer.printLn(`export interface Readable<T> {`);
```

Comment:
Please can you extend `Readable<T>` to include an event for `metadata` and also expose an `abort()` method?